src/pages/patterns/filtering/index.mdx

```diff
@@ -69,6 +69,62 @@ Multiple category selection is most commonly manifested in either a vertical lis
 </Column>
 </Row>
 
+### Advanced filtering
+
+_Advanced search_ can be used when the user requires to search and filter a list of items by choosing more than one filter from multiple filter categories. This pattern is especially useful when the number of items within each category is dynamic, such as filtering a table by a user or user defined tags.
+
+Advanced search is also commonly used when there is not the requried amount of space on the screen to accommodate a vertical list.
+
+<br />
+<Row>
+<Column colLg={8}>
+
+![Advanced filter input field example](images/filter-8.png)
+
+</Column>
+</Row>
+<br />
+<br />
+
+The filtering categories appear as headers within a dropdown space underneath a search field, with the top five items from within each section appearing as defaults within this space, along with the expected number of returns if the user was to select that option.
```

Comment:
what's the significance of the top 5?

Reply:
There is no real significance to the number of items shown, just aware that we need to give an example of the things you can filter by.

If there are a greater number of 'category' of filters then 5 would be quite cumbersome.

Perhaps we can simply change it to "with an example from each section appearing as defaults within this space"